daemon/mgr/container_storage.go

```diff
@@ -345,8 +345,19 @@ func (mgr *ContainerManager) getMountPointFromContainers(ctx context.Context, co
 				return err
 			}
 
-			// the volumes from VolumeFrom is not allowed to CopyData
-			mp.CopyData = false
+			// if mode has no set rw mode, so need to inherit old mountpoint rw mode.
+			if !(strings.Contains(mode, "ro") || strings.Contains(mode, "rw")) {
+				mp.RW = oldMountPoint.RW
+			}
+
+			// if mode has no set copy mode, so need to inherit old mountpoint copy mode.
+			if !strings.Contains(mode, "nocopy") {
```

Comment:
How about:
```
if !strings.Contains(mode, "nocopy") && !strings.Contains(oldMountPoint.Mode, "nocopy") {
    mp.CopyData = true
}
```

`CopyData` default is `false`

Reply:
> How about:
> 
> ```
> if !strings.Contains(mode, "nocopy") && !strings.Contains(oldMountPoint.Mode, "nocopy") {
>     mp.CopyData = true
> }
> ```
> `CopyData` default is `false`

Actually, `opts.ParseBindMode` has changed `CopyData` to true.